example/src/Utilities/payByID.js

```diff
@@ -0,0 +1,29 @@
+import { AdyenCSE, AdyenAction } from '@adyen/react-native';
+import { ENVIRONMENT } from '../Configuration';
+import ApiClient from './APIClient';
+import { checkoutConfiguration } from './AppContext';
+import { isSuccess } from './Helpers';
+
+export async function payByID(id, cvv, configuration) {
+  const encryptedCard = await AdyenCSE.encryptCard(
+    { cvv: cvv },
+    ENVIRONMENT.publicKey
+  );
+  const data = {
+    paymentMethod: {
+      type: 'scheme',
+      storedPaymentMethodId: id,
+      encryptedSecurityCode: encryptedCard.cvv,
+      threeDS2SdkVersion: AdyenAction.threeDS2SdkVersion,
+    },
+  };
+
+  var result = await ApiClient.payments(data, configuration, ENVIRONMENT.returnUrl);
```

Comment:
In general, we should use `let` instead of `var`. 
[this post](https://dev.to/richardbray/why-the-var-and-let-keywords-shouldnt-be-used-interchangeably-3n07) explains much better than I could 🙂.

Reply:
Fixed [97323d1](https://github.com/Adyen/adyen-react-native/pull/342/commits/97323d11bbb6a2b364c930e4265297ee7a598917)